modules/ossm-automatic-gateway-injection.adoc

```diff
@@ -0,0 +1,89 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-traffic-manage.adoc
+
+:_content-type: CONCEPT
+[id="ossm-automatic-gateway-injection_{context}"]
+= Enabling gateway injection
+
+Gateway configurations are applied to standalone Envoy proxies that are running at the edge of the mesh, rather than sidecar Envoy proxies running alongside your service workloads. Because gateways are Envoy proxies, you can configure {SMProductShortName} to automatically inject gateways similar to how you can automatically inject sidecars.
+
+Using automatic injection for gateways lets you deploy and manage gateways independent from the `ServiceMeshControlPlane` resource. This lets you manage your gateways with your user applications. Using auto-injection for gateway deployments gives developers full control over the gateway deployment, while also simplifying operations. When a new upgrade is available, or a configuration has changed, you restart the gateway pods to update them. This makes the experience of operating a gateway deployment the same as operating sidecars.
```

Comment:
```suggestion
Using automatic injection for gateways, you can deploy and manage gateways independent from the `ServiceMeshControlPlane` resource and therefore manage these gateways with your user applications. Using auto-injection for gateway deployments gives developers full control over the gateway deployment while also simplifying operations. When a new upgrade is available or a configuration has changed, you restart the gateway pods to update them. Doing so makes the experience of operating a gateway deployment the same as operating sidecars.
```
- Per the ISG `Avoid stating that inanimate objects grant abilities to people, as in "the product lets you ...". Whenever possible, use a direct, user-focused alternative such as "you can use the product to ..." or "with this product, users can ...".`  Also "This" by itself at the beginning of the second and last sentences is missing an antecedent. Generally, "this" should always be followed by a noun or noun phrase "this something" to avoid unclear antecedents. I've made a couple of suggested edits to avoid this issue, but feel free to work around it by rephrasing in other ways.
- omit unnecessary commas

Reply:
changes made